### DISCUSSION TITLE
Faster to build CRUD APIs?

### DISCUSSION THREAD

**Discussion by mroushdy** [blitz-js/blitz#349] [General] [answered]
Does Blitz make it faster to build CRUD APIs or is it mostly for web?

**Comment by MrLeebo** [comment]
Good question! With blitz, you can code as if you have a function with direct access to your database in your React app. During compilation Blitz will take that data fetching function and generate an RPC endpoint around it. So you can focus on building your app while blitz generates the API layer for you.

**Comment by mroushdy** [comment]
My question was more about using blitz to build a backend for an iOS app

**Comment by Skn0tt** [comment]
Blitz will add support for React Native development in the future, but building backends is out of scope for this. You can read more about this in the [Blitz Manifesto](https://github.com/blitz-js/blitz/blob/canary/MANIFESTO.md#2-api-not-required).

**Comment by Skn0tt** [comment]
If you've got any more questions on Blitz usage, feel free to come by to [our Slack](https://github.com/blitz-js/blitz#welcome-to-the-blitz-community-)! ☺️

**Comment by flybayer** [comment, marked as answer]
@mroushdy yeah, currently the answer to your question is "no". But this is a key future feature for us.

There will be two ways to use Blitz from a mobile app:

1. Direct integration with the same "no API" experience
2. Use an RPC client to access all the same Blitz queries/mutations but without the direct integration. This could be generated with full TS types.
